### PR TITLE
Fix/fetch response error handling

### PR DESCRIPTION
**Pull request by paghar**
- [x] I have followed the [Your First Code Contribution section of the Contributing guide](https://github.com/themesberg/flowbite-react/blob/main/CONTRIBUTING.md#your-first-code-contribution)

Summarize the changes made and the motivation behind them.

Reference related issues using `#` followed by the issue number.

If there are breaking API changes - like adding or removing props, or changing the structure of the theme - describe them, and provide steps to update existing code.

### CODE REVIEW

**Comment by paghar** [comment]
@rluders Hi
Thank you for checking the changes, I will always be happy to help.

File: src/helpers/http.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export const safeResJson = (res: Response) => {
+  if (res.ok) {
+    return res.json()
+  } else {
```
Comment (rluders) [inline src/helpers/http.ts:4]: You don't need the `else` statement. Just throw the exception after the `if` block.

**Review by rluders — CHANGES_REQUESTED** [review]
I'm unsure if I understood what it is trying to solve, but... I guess it doesn't hurt to get it merged after the change.

File: src/helpers/http.ts (hunk 1 above)
Comment (paghar) [reply src/helpers/http.ts]: you are right.
fixed now👌

**Comment by rluders** [comment]
Looks good @paghar. But the prettier doesn't seem happy.

**Comment by paghar** [comment]
@rluders Sorry about that.
I fixed the problem, but I'm unable to update branch with rebase because of a conflict!
So I merged It....Is that OK?

**Comment by rluders** [comment]
> @rluders Sorry about that. I fixed the problem, but I'm unable to update branch with rebase because of a conflict! So I merged It....Is that OK?

You need to rebase... If you are having issues with the rebase, I can try to help tomorrow, just let me know if you didn't get into solving the conflict when rebasing.

**Comment by paghar** [comment]
@rluders It is fixed now.
please check that.

**Review by rluders — APPROVED** [review]

**Pull request merged**